index.html

```diff
@@ -47,97 +48,40 @@ <h2 class="connect_wrapper"><a class="contact" href="#">LETS CONNECT</a></h2>
     </section>
     
     <section class="grid__wrapper" id="portfolio">
-        <div class="covers">
-            <img class="img__class" id="pic__2" src="./img/pic1.png" alt="">
-            <img class="img__class" id="pic__1" src="./img/pic1.png" alt="">
-            <div class="right__box">
-                <h1 class="bold-text">Tonic</h1>
-                <ul class="selections">
-                    <li class="main__txt__wrapper"><a href="#" class="main__txt"></a>CANOPY</li>
-                    <li class="circle__wrapper"><a href="#" class="circle"></a></li>
-                    <li class="list__wrapper"><a href="#" class="list">Back End Dev</a></li>
-                    <li class="circle__wrapper"><a href="#" class="circle"></a></li>
-                    <li class="list__wrapper"><a href="#" class="list">2015</a></li>
-                </ul>
-                <p class="signup-disc">A daily selection of privately personalized reads: no acounts or signup's required</p>
-                <ul class="platform">
-                    <li class="pla__btn">html</li>
-                    <li class="pla__btn">css</li>
-                    <li class="pla__btn">javascript</li>
-                </ul>
-                <button class="see-project">See Project</button>
-            </div>
-        </div>
 
-        <div class="covers">
-            <img class="img__class" id="pic__3" src="./img/pic2.png" alt="">
-            <div class="right__box">
-                <h1 class="bold-text">Multi-Post Stories</h1>
-                <ul class="selections">
-                    <li class="main__txt__wrapper"><a href="#" class="main__txt"></a>FACEBOOK</li>
-                    <li class="circle__wrapper"><a href="#" class="circle"></a></li>
-                    <li class="list__wrapper"><a href="#" class="list">Back End Dev</a></li>
-                    <li class="circle__wrapper"><a href="#" class="circle"></a></li>
-                    <li class="list__wrapper"><a href="#" class="list">2015</a></li>
-                </ul>
-                <p class="signup-disc">A daily selection of privately personalized reads: no acounts or signup's required</p>
-                <ul class="platform">
-                    <li class="pla__btn">html</li>
-                    <li class="pla__btn">css</li>
-                    <li class="pla__btn">javascript</li>
-                </ul>
-                <button class="see-project">See Project</button>
```

Comment:
Pop-up for mobile is missing technologies list and buttons. It does not follow the design. Please see the screenshot below for reference and make it exact same as in the design
![mobile-popup](https://user-images.githubusercontent.com/35504149/136222595-11aecea5-8a3b-4e9a-b7a2-7ee6945c4d44.png)
.